### PR TITLE
Edit get-started/install page

### PR DESCRIPTION
**Pull request by LarryUllman**
This cleans up the get-started/install page 

- change title to "Install"
- page doesn't discuss configuration so I removed that text
- DB content goes on the self-hosted page (makes this page simpler and encourages "just use Cloud")
- remove esoteric content
- make the TOC cleaner and better targeted
- point to quickstart after this instead of the tutorial

I also removed the instructions for how to install SQLite entirely because:

- As mentioned in the text, SQLite is already commonly installed
- That information isn't specific to Prefect
- That information is already--and better--found elsewhere online

### Checklist

- [x] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.

For documentation changes:

- [x] This pull request includes redirect settings in `mint.json` for files that are removed or renamed.

### CODE REVIEW

File: docs/3.0rc/cloud/self-host.mdx [hunk 1]
```diff
@@ -1,16 +1,66 @@
...
+
+Prefect Cloud and a self-hosted Prefect server instance share a common set of capabilities. Prefect Cloud has these additional features:
+
+- [Workspaces](/3.0rc/cloud/workspaces/)&mdash;isolated environments to organize your flows, deployments, and flow runs
+- [Automations](/3.0rc/react/understand-events/automations-triggers/)&mdash;configure triggers, actions, and notifications in response to real-time monitoring events
+- [Email notifications](/3.0rc/react/understand-events/automations-triggers/)&mdash;send email alerts from Prefect's servers based on automation triggers
+- [Service accounts](/3.0rc/cloud/manage-users/service-accounts/)&mdash;configure API access for running workers or executing flow runs on remote infrastructure
+- [Custom role-based access controls (RBAC)](/3.0rc/cloud/manage-users/manage-roles/)&mdash;assign users granular permissions to perform activities within an account or workspace
+- [Single Sign-on (SSO)](/3.0rc/cloud/manage-users/configure-sso/)&mdash;authentication using your identity provider
+- [Audit Log](/3.0rc/cloud/manage-users/audit-logs/)&mdash;a record of user activities to monitor security and compliance
```
Comment (discdiver) [inline docs/3.0rc/cloud/self-host.mdx:19]: Should this be plural?

File: docs/3.0rc/cloud/self-host.mdx [hunk 2]
```diff
@@ -1,16 +1,66 @@
...
+
+## Prefect server installation notes
+
+Your self-hosted server must meet certain requirements and be properly configured. 
+
+### SQLite
+
+Hosting your own Prefect server instance requires a PostgreSQL database, but you can use SQLite 2.24 or newer for development.
```
Comment (discdiver) [inline docs/3.0rc/cloud/self-host.mdx:27]: Just noting that we might change the guidance here. @billpalombi is up on the latest thinking. Don't let potential changes here hold back merging.

File: docs/3.0rc/cloud/self-host.mdx [hunk 3]
```diff
@@ -46,12 +82,13 @@ Go to your terminal session and run this command to set the API URL to point to
 prefect config set PREFECT_API_URL="http://127.0.0.1:4200/api"
 ```
 
-!!! tip "`PREFECT_API_URL` required when running Prefect inside a container"
-    You must set the API server address to use Prefect within a container, such as a Docker container.
+<Info>
+You must set the API server address--`PREFECT_API_URL`--to use Prefect within a container, such as a Docker container.
```
Comment (discdiver) [inline docs/3.0rc/cloud/self-host.mdx:86]:
```suggestion
You must set the API server address—`PREFECT_API_URL`—to use Prefect within a container, such as a Docker container.
```

Noting that our use of em dash style variations is a thing. Let's standardize on [Google style](https://developers.google.com/style/dashes#:~:text=After%20the%20Compose%20key%20is,Then%20press%20Return%20.). Their rec is:

> Em dashes
To indicate a break in the flow of a sentence—or an interruption—use an em dash, also known as a long dash. Don't put a space before or after it.

You can type the em dash character in various ways:

HTML
&mdash;

macOS
Press Option+Shift+hyphen.

**Review by discdiver — COMMENTED** [review]
Nice changes to simplify! Minor nits.

File: docs/3.0rc/cloud/self-host.mdx (hunk 2 above)
Comment (billpalombi) [reply docs/3.0rc/cloud/self-host.mdx:27]: Great callout. We can remove this line entirely.

File: docs/3.0rc/cloud/self-host.mdx (hunk 1 above)
Comment (LarryUllman) [reply docs/3.0rc/cloud/self-host.mdx:19]: Excellent catch. Thanks!

**Review by billpalombi — APPROVED** [review]
lgtm

**Review by discdiver — APPROVED** [review]

**Pull request merged**